tock/hours/forms.py

```diff
@@ -74,6 +74,15 @@ def render_option(self, selected_choices, option_value, option_label):
             conditional_escape(force_text(option_label)))
 
 
+def choice_label_for_project(project):
+    """
+    Returns the label for a project as it should appear in an
+    auto-completable list of choices.
+    """
+
+    return '%s - %s' % (project.id, project.name)
```

Comment:
I also decided to factor `choice_label_for_project` out as a function so that tests could use it; that way if we decide to change the labeling in the future, we just change it in the function and the function's test, rather than causing regressions in other unrelated tests.